src/common.d.ts

```diff
@@ -1,11 +1,15 @@
 export interface Tokens {
-  accessToken: string;
-  refreshToken: string;
-  canvasUserId: string;
+  access_token?: string,
+  access_token_secret?: string, 
+  token_type?: string, 
+  expires_in?: Number, 
+  refresh_token?: string,
+  lastRefresh?: Date,
+  info?: any,
 }
 
-export type GetUserToken = (userId: string) => Promise<Tokens>;
-export type SetUserToken = (userId: string, tokens: Tokens) => Promise<void>;
+export type GetAccessToken = (userId: string) => Promise<Tokens>;
+export type SetAccessToken = (userId: string, token: Tokens) => Promise<any>;
```

Comment:
what does setAccessToken actually return ? earlier it was void that why asking, are we returning anything now ?

Reply:
yes, i checked the code, with setToken, we return the whole of the user object, as per the current contract, void was wrong previously. This is just type definition, and not implementation change, hence did not break.